addons/arsenal/script_component.hpp

```diff
@@ -5,6 +5,7 @@
 // #define DEBUG_MODE_FULL
 // #define DISABLE_COMPILE_CACHE
 // #define ENABLE_PERFORMANCE_COUNTERS
+// #define ENABLE_PERF_PROFILING
```

Comment:
maybe add a little note about what this does / requires

Reply:
true.